### PR TITLE
re-added the files to the folder under a different branch

### PR DESCRIPTION
**Pull request by borja-x13**
# Submitting Week 3 Assessment

### ✅ Completed Files

Mark the following as confirmation that the files in this repository are complete.

- [ X] Coding Challenges
- [ X] Interview Questions
- [ X] React Debugging

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: interview-questions.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Your answer: DOM or document object model is a sort of reversed UI, whereas the UI makes the code readable to humans, the DOm makes it interactable to the computer. An API for the webpage. 
+
+Researched answer: The DOM makes the HTML document more dynamic as it views the document as a set of nodes allowing us to change the nodes by creating events dynamically. 
+
+6. STRETCH: Which is the difference between a div tag and a span tag?
+
+Your answer: A div tag and a span tag are both a way to create a "block" to separate html/js code for organization and styling purposes. The difference between them is strictly semantical in nature.
+
+Researched answer: Div tags are block-level elements and are used to wrap sections of a document for styling. Span tags are inline elements used for styling sections of text or images. 
+
```
Comment (triciasykes) [inline interview-questions.md:44]: great job on these answers, Borja! I encourage you to use the research to learn something you didn't already know (i.e. your answer was correct :) ).

File: react-debugging.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      }
+
+
+      // No need to change anything below this line
+
+      const domContainer = document.querySelector('#react-container')
+      const root = ReactDOM.createRoot(domContainer)
+      root.render(<App />)
+    </script>
+  </body>
```
Comment (triciasykes) [inline react-debugging.html:96]: Nice work finding those bugs!

File: w3- code-challenges.test.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+      secondFib = nextFib 
+    }
+  }
+  return fibArray
+}
+
+// output PASS  ./w3- code-challenges.test.js
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
```
Comment (triciasykes) [inline w3- code-challenges.test.js:88]: Excellent Borja!  Your pseudocode and test are great.  I like the way you set up your function. 
Just for fun, see if you can refactor with higher order functions. 😬

File: w3- code-challenges.test.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+  let valuesArray = Object.values(object)
+  valuesArray = valuesArray.sort((a, b) => a - b)
+  return valuesArray
+}
+
+// output  
+// PASS  ./w3- code-challenges.test.js
+// Test Suites: 1 passed, 1 total
+// Tests:       2 passed, 2 total
+
```
Comment (triciasykes) [inline w3- code-challenges.test.js:161]: beautiful code. Very elegant.

File: w3- code-challenges.test.js [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+// input: array of numerical values
+// output: array of the same length with accumulated sum values
+// expected result:  PASS
+
+const accumulatedSumArray = (array) => {
+  let sum = 0
+  let accumulatedArray = array.map(value => sum += value)
+  return accumulatedArray
+}
+
```
Comment (triciasykes) [inline w3- code-challenges.test.js:217]: Amazing pseudocode!  Again fantastic code, Borja!

**Review by triciasykes — APPROVED** [review]
Great work Borja! Your pseudocoding skills are impressive!  Excellent, clean code.  The only thing I'd encourage is to put a little more into your interview questions.  It's more than just knowing the definition, so using examples/experience will really show off your understanding to your interviewers. This is just a tip, you're doing awesome!  Excellent work overall - you're coding at a Competent Level!

File: interview-questions.md (hunk 1 above)
Comment (borja-x13) [reply interview-questions.md:44]: LOL I did learn, I didn't realize that a span was inline level. I thought they were both block level. I think I didn't convey the learning part very well. Thank you Trish.

**Pull request merged**